src/app/components/blocks/Sidebar.tsx

```diff
@@ -65,43 +65,40 @@ const SidebarBlock: FC<SidebarBlockProps> = ({ links, className }) => {
 
   return (
     <div className={classNames("flex flex-col", className)}>
-      {links.map(({ route, label, Icon, action, soon }) => {
+      {links.map(({ route, label, Icon, action, soon, badge }) => {
         const isPageActive = route === page;
         const notificationBadge = route === Page.Settings && updateAvailable;
 
         if (typeof action === "function") {
           return (
-            <Button
-              theme="clean"
+            <button
               key={label}
               onClick={action}
               className={classNames(
                 "group",
                 "text-base !font-bold text-brand-light/80",
                 "w-52 !py-2 !px-3 !mb-2",
                 "rounded-[.625rem]",
-                "flex !justify-start",
-                "transition-colors",
-                "group",
-                "hover:text-brand-light",
-                "focus:text-brand-light",
-                isPageActive && "bg-brand-main/5 !text-brand-light",
+                "flex justify-start items-center",
+                isPageActive && "!bg-brand-main/5 !text-brand-light",
                 "last:mb-0",
+                "hover:text-brand-light",
               )}
             >
-              <Icon
-                className={classNames(
-                  "w-7 h-7",
-                  "min-w-7",
-                  "mr-5",
-                  "glass-icon",
-                  isPageActive && "glass-icon--active",
-                  "group-hover:glass-icon--hover",
-                  "group-focus:glass-icon--hover",
-                )}
-              />
-              {label}
-            </Button>
+              <BadgeWrapper showBadge={badge}>
+                <Icon
+                  className={classNames(
+                    "w-7 h-7",
+                    "min-w-7",
+                    "styled-icon",
+                    isPageActive
+                      ? "styled-icon--active"
+                      : "group-hover:styled-icon--hover group-focus:styled-icon--hover",
+                  )}
```

Comment:
The the code as used for link. Try to reuse shared code, for example:
```ts
const Component = isButton ? "button" : Link;

return (<Component {...props} />)
```

Reply:
Well, I moved easy-to-reuse code into <LinkContent /> and provided it as button's child.
I don't think either Link and button has many common props to to smth like `<Component {...props} />`